### PR TITLE
Possibly show Price in Minor charter

### PR DESCRIPTION
**Pull request by perwestling**
Show price if minor has a value and no owner.

### CODE REVIEW

**Comment by perwestling** [comment]
If the minor has attribute 'value' non-zero, do show Price in charter:
image

If discount is non-zero, present Price as value - discount.

If value is zero, or owner is set, do not change current view. So it will looks like this:
image

**Comment by perwestling** [comment]
Commit also contains some updates to use the new value and discount attributes. As far as I know it is so far only 1893 that uses these.

File: assets/app/view/game/corporation.rb [hunk 1]
```diff
@@ -311,6 +313,11 @@ def render_tokens
         end)
       end
 
+      def render_minor_price
+        price_str = "Price: #{@game.format_currency(@corporation.value - @corporation.discount)}"
```
Comment (tobymao) [inline assets/app/view/game/corporation.rb:317]: should this be min_bid?

**Comment by tobymao** [comment]
why do minors have value and discount? shouldn't they be companies then?

File: assets/app/view/game/corporation.rb (hunk 1 above)
Comment (perwestling) [reply assets/app/view/game/corporation.rb]: Fixed

**Comment by perwestling** [comment]
> why do minors have value and discount? shouldn't they be companies then?

@tobymao Minors is in between Company and Corporation.
They do operate - but does not have any shares - but they have an owner.
They cannot be sold. They can have abilities. etc

In latter titles (1893, and maybe 1824) I have moved towards buying the minor instead of buying a company which is connected to a minor and indirectly taken over it.  

So I introduced possibility to buy Minor during the auction. 
But the Minor did not show the value/cost. If you clicked on them a Buy button appear, but you cannot do this for all.

In all it makes for an easier implementation, to use Minors directly instead of Proxy privates.

**Comment by tobymao** [comment]
but don't all other games like 1846 use company and then convert to minor?

**Comment by perwestling** [comment]
Ok, I will throw the changes with minors out, by making a commit that uses privates instead.

**Pull request closed without merging**